src/pages/{storiesJson.title}.jsx

```diff
@@ -280,6 +286,7 @@ export const query = graphql`
         media
         sentence
         word
+        definition
```

Comment:
Doesn't look like this is getting used. Can we just remove it?

Reply:
This also got resolved in the TTS branch.